frontend/simple/views/Invite.vue

```diff
@@ -110,8 +116,12 @@ export default {
           contract.constructor.vuex.mutations[type](state, action.data)
         })
         let mailbox = await backend.latestHash(state.attributes.mailbox)
-        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId}, mailbox)
+        let date = new Date()
+        let invite = new Events.PostInvite({groupId: this.$store.state.currentGroupId, sentDate: date.toString()}, mailbox)
         await backend.publishLogEntry(state.attributes.mailbox, invite)
+        let latest = await backend.latestHash(this.$store.state.currentGroupId)
+        let invited = new Events.RecordInvitation({ username: member.name, inviteHash: invite.toHash(), sentDate: date.toString() }, latest)
+        await backend.publishLogEntry(this.$store.state.currentGroupId, invited)
```

Comment:
There's zero error handling here. What happens if any of these `awaits` fail? We're at the top-level, therefore needs a try-catch.